config/initializers/pipeline.rb

```diff
@@ -0,0 +1,19 @@
+module IdSeqPipeline
+  AWS_ACCESS_KEY_ID = ENV['AWS_ACCESS_KEY_ID']
+  AWS_SECRET_ACCESS_KEY = ENV['AWS_SECRET_ACCESS_KEY']
+  AWS_DEFAULT_REGION = ENV['AWS_DEFAULT_REGION']
+  S3_SCRIPT_LOC= 's3://czbiohub-infectious-disease/bin/pipeline.py'
+  BASE_COMMAND = "export AWS_ACCESS_KEY_ID=#{AWS_ACCESS_KEY_ID}" +
+                 " AWS_SECRET_ACCESS_KEY=#{AWS_SECRET_ACCESS_KEY}" +
+                 " AWS_DEFAULT_REGION=#{AWS_DEFAULT_REGION}; "
+  def self.pipeline_command(input_s3_dir, output_s3_dir, db_sample_id)
```

Comment:
this code should not be in initializers

Reply:
where should it go?

Reply:
probably in the job

Reply:
you can forget about the other notes, this is the only part I think we should change now (vs later PRs)